### PR TITLE
added "add item" button and fix margin issue

### PR DESCRIPTION
**Pull request by oscarchong**
Before there's a UI issue on all the pages where there's extra white space when slide scrolled to the right. 

This PR fixed that issue. Additionally, I added an "Add Item" button on nav bar, this allows user to add/upload an item anywhere(when they're on product or profile page) besides just the homepage.

**Before:**
Screen Shot 2019-12-10 at 11 03 49 PM

**After:**
Screen Shot 2019-12-10 at 11 04 07 PM

### CODE REVIEW

**Comment by Akasora39** [comment]
Do you know if this fixes it for all pages? Because I know the same issue happened for the detailed product page as well.

**Comment by oscarchong** [comment]
> Do you know if this fixes it for all pages? Because I know the same issue happened for the detailed product page as well.

Yea, this will be fixed for all the pages

File: client/src/js/components/Navbar/SignedIn.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Enzyme, { shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { MemoryRouter } from 'react-router-dom';
+import { SignedIn } from './SignedIn';
```
Comment (Akasora39) [inline client/src/js/components/Navbar/SignedIn.test.js]: Looks like you imported the unconnected component but you're connecting to a mock store so you probably want to either use the connected component or don't use a mock store.

Comment (oscarchong) [reply client/src/js/components/Navbar/SignedIn.test.js]: oh alright i was trying out a lot of stuff but forgot to remove them

**Comment by Akasora39** [comment]
> The test itself looks fine. I think it's a matter of how you imported the component.

Disregard this. Looks like it's moreso an issue regarding testing React hooks. I'll take a look at it.

**Pull request merged**